src/address.rs

```diff
@@ -10,10 +10,10 @@
 
 //! Traits to represent an address within an address space.
 //!
-//! Two traits are defined to present an address within an address space:
+//! Two traits are defined to represent an address within an address space:
 //! - [AddressValue](trait.AddressValue.html): stores the raw value of an address. Typically u32,
```

Comment:
```suggestion
//! - [AddressValue](trait.AddressValue.html): stores the raw value of an address. Typically `u32`,
//! `u64` or `usize` is used to store the raw value. But pointers, such as `*u8`, can't be used because
//! they don't implement the [`Add`](https://doc.rust-lang.org/std/ops/trait.Add.html) and
//! [`Sub`](https://doc.rust-lang.org/std/ops/trait.Sub.html) traits.
```

Reply:
Fixed.